src/libs/actions/IOU.js

```diff
@@ -625,6 +625,7 @@ function buildOnyxDataForMoneyRequest(
  * @param {Object} [policy]
  * @param {Object} [policyTags]
  * @param {Object} [policyCategories]
+ * @param {Number} [moneyRequestReportID]
```

Comment:
NAB - would it be helpful to explain the purpose of this new param? I'm not exactly sure how to explain, i tried for like 5 minutes 🤣 

Like how can we explain why `chatReport.iouReport` is useful sometimes but not always, which is why we need this param? Or is that obvious to everyone but me? :D

Reply:
Yeah I was actually thinking about the same but did not come up with anything short. I am adding something now, let me know how it sounds!